components/tag-list/tag-list-item.js

```diff
@@ -0,0 +1,25 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { TagListItemMixin } from './tag-list-item-mixin.js';
+
+class TagListItem extends TagListItemMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: Text to display
+			 * @type {string}
+			 */
+			text: { type: String }
+		};
+	}
+
+	render() {
+		const tag = this._renderTag(this.text);
```

Comment:
I think you can just return `this._renderTag(this.text)`, no need to wrap the html in another html.